packages/vaadin-notification/src/vaadin-notification.js

```diff
@@ -388,7 +383,8 @@ class NotificationElement extends ThemePropertyMixin(ElementMixin(PolymerElement
     this._oldRenderer = renderer;
 
     if (rendererChanged) {
-      card.innerHTML = '';
+      this._card = this._card || this.shadowRoot.querySelector('vaadin-notification-card');
```

Comment:
Previously, `this._card` used `this.$['vaadin-notification-card']`. This method stores the card element before any effects would be called, so that will keep the ref to the card element even it is moved to the notification container.

As we have decided to remove the ID attribute, we can't access the card element through `this.$.` anymore. This is why this workaround arose.

Reply:
This is really tricky to time properly. Another approach might be to keep the previous logic but manually remove the id attribute from `this._card` in `ready()`...not sure if it's any cleaner though.

Reply:
Good point. Thanks! 

All the ways looks a kind of dirty, but the removing the ID attribute approach requires fewer changes. 

Even though it is not the most obvious way either (the ID attribute is defined in the template, but doesn't really exist after initialization), I think it isn't a big deal for the `20.0` branch. In the `master` branch we don't have this problem.

So, I'm inclined to use the suggested approach instead.